src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -206,10 +208,13 @@
 import seedu.address.commons.core.index.Index;
 import seedu.address.logic.Messages;
 import seedu.address.logic.commands.EditCommand.EditPersonDescriptor;
+import seedu.address.model.AddressBook;
 import seedu.address.model.Model;
 import seedu.address.model.ModelManager;
 import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
 import seedu.address.testutil.EditPersonDescriptorBuilder;
+import seedu.address.testutil.PersonBuilder;
```

Comment:
doesnt calling the PersonBuilder throw an error?

Reply:
yep changed it to patientbuilder instead